drivers/CoreIMU/source/CoreLSM6DSOX.cpp

```diff
@@ -105,4 +119,17 @@ auto CoreLSM6DSOX::ptr_io_read(CoreLSM6DSOX *handle, uint8_t read_address, uint8
 	return handle->read(read_address, number_bytes_to_read, p_buffer);
 }
 
+void CoreLSM6DSOX::setGyrDataReadyInterrupt()
+{
+	lsm6dsox_pin_int1_route_t gyro_int1 {.drdy_xl = PROPERTY_ENABLE, .den_flag = PROPERTY_ENABLE};
+	lsm6dsox_dataready_pulsed_t drdy_pulsed {LSM6DSOX_DRDY_PULSED};
+
+	lsm6dsox_data_ready_mode_set(&_register_io_function, drdy_pulsed);
+	lsm6dsox_pin_int1_route_set(&_register_io_function, gyro_int1);
```

Comment:
it's usually easier to read when instantiation and use are close to each other:

```suggestion
	lsm6dsox_pin_int1_route_t gyro_int1 {
		.drdy_xl  = PROPERTY_ENABLE,
		.den_flag = PROPERTY_ENABLE,
	};
	lsm6dsox_pin_int1_route_set(&_register_io_function, gyro_int1);
	
	lsm6dsox_data_ready_mode_set(&_register_io_function, LSM6DSOX_DRDY_PULSED);
```